lib/generators/scenic/view/view_generator.rb

```diff
@@ -8,6 +8,10 @@ class ViewGenerator < Rails::Generators::NamedBase
       source_root File.expand_path("../templates", __FILE__)
 
       def create_view_definition
+        unless views_path.exist?
```

Comment:
Yes, I thought to my self that there should be a better name. I like create_views_directory or create_db_views_directory.